### PR TITLE
Update binding labels

### PR DESCRIPTION
**Pull request by lindydonna**
- DocumentDb -> Cosmos DB
- Event Hub -> Event Hubs
- Misc label cleanup

### CODE REVIEW

**Comment by lindydonna** [comment]
Fixes #445

**Comment by lindydonna** [comment]
@soninaren Please review and merge.

File: Functions.Templates/Resources/Resources.resx [hunk 1]
```diff
@@ -1096,16 +1096,16 @@
...
   <data name="documentDB_displayName" xml:space="preserve">
-    <value>Azure DocumentDB Document</value>
+    <value>Azure Cosmos DB</value>
   </data>
   <data name="sendGrid_apiKey_help" xml:space="preserve">
     <value>The name of the app setting containing your SendGrid developer API key.</value>
   </data>
   <data name="sendGrid_apiKey_label" xml:space="preserve">
-    <value>SendGrid API Key</value>
+    <value>SendGrid API Key App Setting</value>
```
Comment (soninaren) [inline Functions.Templates/Resources/Resources.resx]: nit: SendGrid API Key App Setting Name

**Comment by soninaren** [comment]
Please verify the labels on next portal. Updates every hour

**Pull request merged**